### PR TITLE
Aligned buttons and dash

### PR DESCRIPTION
**Pull request by LeeRenJie**
## Issue that this pull request solves

 Closes: #369 

## Proposed changes
- [x] Fix the join us button on the home page
- [x] Fix the width of the dash under the HIT Tech Community title of the home page
- [x] Fix Subscribe to newsletter section (the input box, its alignment, etc.)
- [x] Fix the All Broadcasts button on the Broadcasts page(It should appear in center)
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes do not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of a change in the user interface
- Join Us Button centered
image

-  Fixed dash under title
image

- Subscribe to the newsletter box aligned
image

-  All Broadcast button centered
image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by LeeRenJie** [comment]
I am under DWOC. 😄 Any changes needed please state, Thanks!

File: frontend/src/components/home/overview/overview.module.css [hunk 1]
```diff
@@ -27,7 +27,7 @@
 }
 
 .dash {
-  width: 90px;
+  width: 450px;
```
Comment (Kajol-Kumari) [inline frontend/src/components/home/overview/overview.module.css:30]: let's keep `width:300px`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please make the footer's newsletter section's input field and submit button's height `35px`

Rest looks good!

File: frontend/src/components/home/overview/overview.module.css (hunk 1 above)
Comment (LeeRenJie) [reply frontend/src/components/home/overview/overview.module.css]: Updated:
image
image

Comment (LeeRenJie) [reply frontend/src/components/home/overview/overview.module.css]: @Kajol-Kumari I'm done with the edit! Please review thanks! 😄

**Review by tejan-singh — APPROVED** [review]

**Comment by LeeRenJie** [comment]
@tejan-singh the label for DWOC was not added, would it be counted?

**Pull request merged**